### PR TITLE
Normalizing training data

### PR DESCRIPTION
**Pull request by Vages**
Add a simple function for normalization of training data. Can also batch normalize training data files from a glob expression.

### CODE REVIEW

**Comment by Vages** [comment]
Taking the freedom to merge it, as no part of the flow has been changed.
